### ISSUE TITLE
Mismatch of  python version used in voltron/gdb?

### ISSUE THREAD

**Issue by dthierbach** [snare/voltron#135]
I'm using gdb 7.10 on Debian, and that is apparently linked with python 3.5.1. I tried to install voltron with pip (Python 2.7), but that fails on the "import blessed" statement when starting gdb. Then I tried to install voltron with pip3, but here the setup fails.

Which version(s) of Python does voltron work with? The documentation mentions a 2.7 path, so I suppose it doesn't work with Python 3? How can I get gdb to use it, then?

**Comment by uberspot** [comment]
I have the same issue on Arch. Also tried installing with pip3 with no luck. The blessed module can't be found even though I installed both the python2 and python3 version of it.

**Comment by ahjkk** [comment]
me too . in MacOSX

**Comment by snare** [comment]
Works for me on Ubuntu 14.04 with Python 3.3/3.4/3.5, and also on Arch.

https://github.com/snare/voltron/wiki/Voltron-on-Ubuntu-with-GDB

How is the setup.py failing?

**Comment by uberspot** [comment]
Traceback (most recent call last):
	  File "~/voltr/voltron/entry.py", line 58, in <module>
	ImportError: No module named 'voltron.plugin'

	During handling of the above exception, another exception occurred:

	Traceback (most recent call last):
	  File "~/voltr/voltron/entry.py", line 113, in <module>
	AttributeError: module 'blessed' has no attribute 'Terminal'

Arch linux x86_64, python3.5. I did run `sudo python3 setup.py install` first.

**Comment by NikolaMandic** [comment]
guys I am not sure if this is the right place to write this but gdb has ui embedded that voltron tries to remake. It just needs a little bit of turning it on :-D

**Comment by snare** [comment]
I know about TUI. Voltron does plenty that TUI doesn't, and works with other debuggers as well.

**Comment by snare** [comment]
@uberspot did you install from PyPI or GitHub? Could you try the latest source from GitHub please? I tested it on Arch x64/Python 3.5 a couple of days ago and it was working.

**Comment by dthierbach** [comment]
@snare: Ok, I installed from github as described in the Ubuntu link, and it works. At least gdb loads without error now.

To make it install, I had to install a few other packages (python3-requests and python3-flask from Debian via apt-get, scruffington via pip3). 

I'm now also totally confused about python package management on Debian, because apparently pip3 likes to install in ~/.local, while setup.py installs in /usr/local (which I'd prefer), but I guess I can sort this out somehow.
